Assets/HoloToolkit/Sharing/Scripts/Utilities/SharingServiceStatus.cs

```diff
@@ -0,0 +1,61 @@
+﻿using UnityEngine;
+using System.Collections;
+using UnityEngine.UI;
+
+namespace HoloToolkit.Sharing
+{
+    public class SharingServiceStatus : MonoBehaviour
+    {
+        enum Status
+        {
+            Disconnected,
+            Connecting,
+            Connected,
+        }
+
+
+        [Tooltip("Reference to Text UI control where the Status info should be displayed.")]
+        public Text Text;
```

Comment:
I am not opposed to TextMesh. I will have to look again but from what I remember, I've had issues with it in the past specifically around setting a color to the text (known bug, can't find the link right now) - by default it's black so it's not visible on HoloLens. 
Do you happen to have a reference where it's used to I can take a look ?